### PR TITLE
Add Automatic save support in editor

### PR DESCRIPTION
**Pull request by rmongia**
- spawns every 10s
- does not send notifications for auto-save

Issue: https://github.com/nvbn/everpad/issues/231

### CODE REVIEW

File: everpad/pad/editor/__init__.py [hunk 1]
```diff
@@ -191,8 +197,7 @@ def update_title(self):
         self.setWindowTitle(self.tr('Everpad / %s') % title)
 
     @Slot()
-    def save(self):
-        self.logger.debug('Saving note: "%s"' % self.note.title)
```
Comment (wackywendell) [inline everpad/pad/editor/__init__.py:195]: Question: why are you removing the logging line? Is it because its too much output?

Comment (rmongia) [inline everpad/pad/editor/__init__.py:195]: sorry, this seems to have been by mistake. I will update the diff.

**Comment by rmongia** [comment]
how do you propose I make it right? This is my first time trying to
contribute to everpad.

On Thu, Jun 19, 2014 at 12:30 AM, Coveralls notifications@github.com
wrote:

> [image: Coverage Status] https://coveralls.io/builds/880839
> 
> Coverage decreased (-0.04%) when pulling _a4f9ebb
> https://github.com/nvbn/everpad/commit/a4f9ebb3557ae74d5e7bc8f356ef6405e72c8099
> on rmongia:develop_ into _ec84b98
> https://github.com/nvbn/everpad/commit/ec84b9815afb9de1bca66c9f1642bf90de89d833
> on nvbn:develop_.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nvbn/everpad/pull/432#issuecomment-46478700.

## 

Ramandeep Singh
Software Engineer | Instart Logic
(M) - +91-8105173059

**Comment by nvbn** [comment]
Hi, maybe better to make it optional?

**Comment by rmongia** [comment]
It seemed like a must have any note taking application. Although, it does no harm to provide a setting for this. Your call.

**Comment by nvbn** [comment]
Thank you, i'll add option for that later.

**Comment by rmongia** [comment]
Sure, thanks.

On Tue, Jun 24, 2014 at 8:45 PM, Vladimir Iakovlev <notifications@github.com

> wrote:
> 
> Thank you, i'll add option for that later.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nvbn/everpad/pull/432#issuecomment-46984550.

## 

Ramandeep Singh
Software Engineer | Instart Logic
(M) - +91-8105173059

**Pull request merged**